src/classlibnative/bcltype/number.cpp

```diff
@@ -134,8 +135,191 @@ unsigned int Int64DivMod1E9(unsigned __int64* value)
 
 #else // _TARGET_X86_ && !FEATURE_PAL
 
-#pragma warning(disable:4273)
-extern "C" char* __cdecl _ecvt(double, int, int*, int*);
+void ecvt( double value, int count, int* dec, int* sign, wchar_t* digits )
```

Comment:
I thought the Windows implementation is in elsewhere: https://github.com/dotnet/coreclr/blob/32f0f9721afb584b4a14d69135bea7ddc129f755/src/vm/i386/fptext.asm#L78

Are you saying delete Line 137 - 138 will force Windows implementation being replaced by mine?